diff -urN grcm-0.1.5/grcm.desktop grcm-0.1.5.new/grcm.desktop
--- grcm-0.1.5/grcm.desktop	2003-02-21 01:23:30.000000000 +0100
+++ grcm-0.1.5.new/grcm.desktop	2005-03-21 13:52:25.776305376 +0100
@@ -1,7 +1,10 @@
 [Desktop Entry]
+Encoding=UTF-8
 Name=Grcm
 Comment=Remote Connection Manager
 Exec=grcm
-Terminal=0
+Terminal=false
 Type=Application
 Icon=grcm/icon.png
+Categories=Application;Network;
+# vi: encoding=utf-8
\ Brak znaku nowej linii na koñcu pliku
diff -urN grcm-0.1.5/Makefile.am grcm-0.1.5.new/Makefile.am
--- grcm-0.1.5/Makefile.am	2003-02-15 21:59:58.000000000 +0100
+++ grcm-0.1.5.new/Makefile.am	2005-03-21 13:45:13.926956456 +0100
@@ -10,7 +10,7 @@
 	omf.make \
 	grcm.desktop
 
-Applicationsdir = $(datadir)/gnome/apps/Internet
+Applicationsdir = $(datadir)/applications
 Applications_DATA = grcm.desktop
 
 install-data-local:
