diff -urN grcm-0.1.4.orig/Makefile.am grcm-0.1.4/Makefile.am
--- grcm-0.1.4.orig/Makefile.am	Fri Feb 21 14:06:13 2003
+++ grcm-0.1.4/Makefile.am	Fri Feb 21 14:06:58 2003
@@ -10,7 +10,7 @@
 	omf.make \
 	grcm.desktop
 
-Applicationsdir = $(datadir)/gnome/apps/Internet
+Applicationsdir = $(datadir)/applications
 Applications_DATA = grcm.desktop
 
 install-data-local:
diff -urN grcm-0.1.4.orig/grcm.desktop grcm-0.1.4/grcm.desktop
--- grcm-0.1.4.orig/grcm.desktop	Fri Feb 21 14:06:13 2003
+++ grcm-0.1.4/grcm.desktop	Fri Feb 21 14:06:38 2003
@@ -5,3 +5,4 @@
 Terminal=0
 Type=Application
 Icon=grcm/icon.png
+Categories=Application;Network;
